### DISCUSSION TITLE
Suggestion for where to store /copyq/themes that are not being used

### DISCUSSION THREAD

**Discussion by Raytle** [hluk/CopyQ#1705] [Ideas]
Instead of storing themes in, for example, `/usr/share/copyq/themes` why not store them in, for example, `~/.config/copyq/themes/themes_currently_not_being_used` along with a README file that includes a link to https://copyq.readthedocs.io/en/latest/theme.html#style-sheets

Alternatively, why not store themes in `~.config/copyq/themes` with the following names...

1. `Remove_this_text_and_the_three_dashes_to_use_this_theme---dark.ini`
2. `Remove_this_text_and_the_three_dashes_to_use_this_theme---light.ini`
3. `Remove_this_text_and_the_three_dashes_to_use_this_theme---forest.ini`
4. and so on along with a README file that includes a link to https://copyq.readthedocs.io/en/latest/theme.html#style-sheets

**Comment by hluk** [comment]
Directory `/usr/share/copyq/themes` contains installed/default theme configuration - they could change in future versions. The app should not silently override the any files under `~/.config` (user could have already modified the files).

**Comment by Raytle** [comment]
That makes sense.
